doc/source/whatsnew/v0.20.0.txt

```diff
@@ -450,6 +450,26 @@ New Behavior:
 
    df.groupby('A').agg([np.mean, np.std, np.min, np.max])
 
+HDFStore where string comparison
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In previous versions most types could be compared to string column in a ``HDFStore``
+usually resulting in an invalid comparsion.  These comparisions will now raise a
+``TypeError`` (:issue:`15492`)
+
+New Behavior:
+
+.. code-block:: ipython
+
+   df = pd.DataFrame({'unparsed_date': ['2014-01-01', '2014-01-01']})
+   df.to_hdf('store.h5', 'key', format='table', data_columns=True)
+   ts = pd.Timestamp('2014-01-01')
+   try:
```

Comment:
instead of putting the try/except. Just show the actual traceback (well just the exception part), e.g.

I would show like this
```
In [2]: 1/0
ZeroDivisionError: division by zero
```

for a code block I literally execute the code and copy-paste (as it is not executed), just formatted.